### DISCUSSION TITLE
Saleor Demo on Localhost

### DISCUSSION THREAD

**Discussion by Emmanuelmichaelmaro** [saleor/saleor#11633] [Q&A]
I was just trying figure out, how to setup saleor demo on localhost. Because on saleor repo i recently clone i can see DemoGraphqlView class but hasn't called anywhere.

**Comment by karolkielecki** [comment]
Hi @Emmanuelmichaelmaro, could you please share the steps to reproduce your issue?

**Reply by Emmanuelmichaelmaro** [reply]
I have saleor core instance running on my localhost, on the other end i have my storefront that already connected to my localhost which running that's core application. When i take a look at saleor/urls.py i can see it point to normal graphql view but i want it to point to demo graphql view which all data are read-only.

**Reply by Emmanuelmichaelmaro** [reply]
I want all the request from frontend to be served from [https://github.com/saleor/saleor/blob/main/saleor/demo/urls.py](this url)
